mainapp/templates/login.html

```diff
@@ -6,7 +6,12 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-6">
-            <h2>Login</h2>
+            <h2>Login <button onclick="goBack()">Back</button></h2>
```

Comment:
near the title doesn't seem to be a great place to have the back button, also including it inside the `<h2>` tag makes it super big.  Please move it down to the bottom of the page, opposite from the login button.  Also if you run the code you can see that the back and login buttons do not look the same, please match the style so they look consistent.